key_test.go

```diff
@@ -0,0 +1,35 @@
+package myradio
+
+import (
+	"testing"
+)
+
+func TestGetApiKeyFromFileWorks(t *testing.T){
```

Comment:
Merge all these test cases into a single function with a list & anonymous struct?
Also, testing a key with spaces in it? (Not sure whether that should pass or fail)